artifactory/cli.go

```diff
@@ -1042,7 +1043,18 @@ func mvnCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	mvnCmd := mvn.NewMvnCommand().SetConfiguration(buildConfiguration).SetConfigPath(configFilePath).SetGoals(filteredMavenArgs).SetThreads(threads).SetInsecureTls(insecureTls).SetDetailedSummary(detailedSummary).SetXrayScan(xrayScan)
+	filteredMavenArgs, format, err := coreutils.ExtractXrayOutputFormat(filteredMavenArgs)
+	if err != nil {
+		return err
+	}
+	if xrayScan && format != "" {
+		return cliutils.PrintHelpAndReturnError("--format flag is valid only if --scan is true", c)
```

Comment:
--format flag is valid only if --scan is true
-->
The --format option can be sent only with the --scan option

The same applies to the ```gradleCmd``` function.